packages/system/src/generator.ts

```diff
@@ -21,7 +21,21 @@ export class StyleGenerator {
     const pseudoProps: { [key: string]: any } = {};
 
     for (const [propName, propValue] of Object.entries(props)) {
-      if (isStyledProp(propName.trim())) {
+      if (
+        typeof propValue === "string" &&
+        /[a-zA-Z]+\.[a-zA-Z0-9]+/.test(propValue)
+      ) {
+        const userTheme = theme.getUserTheme();
+        const propKey = propValue.split(".")[0] as StyleThemeKeyType;
+        if (userTheme[propKey] !== undefined) {
+          for (const key in userTheme[propKey]) {
+            if (propValue.trim() === key) {
+              styledProps[propName.trim()] = userTheme[propKey]![key];
+              break;
+            }
+          }
+        }
+      } else if (isStyledProp(propName.trim())) {
```

Comment:
Themes like `fonts` and `spacings` are increasing, and we plan to add various themes in the future as well.
As there are cases where `converter` differ for each theme token, it's not practical to process theme values all together here.
Could you please revert the changes to this code? :pray:

Reply:
Sorry, please wait. I checked your Discord message. I will review again after confirming.

Reply:
This code doesn't seem to need to be reverted, so I resolve conversation.